charts/incubator/owntracks/values.yaml

```diff
@@ -0,0 +1,33 @@
+image:
+  repository: tccr.io/truecharts/owntracks
+  pullPolicy: IfNotPresent
+  tag: 0.9.2@sha256:c774c7db9f80da21e11599872f9c4bdfdc7316834bd88685f4a0e3d3cbc915fa
```

Comment:
this container is not yet available, change this to the upstream for initial testing.

Reply:
Thank you for the direction. I see that the tag has been updated, should I still switch over to docker hub?